bot/events.go

```diff
@@ -75,13 +75,6 @@ func eventsAPIHandler(botCtx cmd.BotContext) http.HandlerFunc {
 				}
 
 				switch event.Channel {
-				case channelStaff:
-					if event.User == "USLACKBOT" {
-						m := inviteUsersMessageRegex.FindStringSubmatch(event.Text)
```

Comment:
You must also remove the regex var (`inviteUsersMessageRegex`)

Reply:
In fact, its so weird because the app was built successfully by gh actions which thats not expected when a var is declared and not used...

Reply:
I will reject this PR and create a new one with a proper branch and removing the code not used